docs/self-managed/identity/troubleshooting/common-problems.md

```diff
@@ -0,0 +1,55 @@
+---
+id: common-problems
+title: "Common Problems"
+sidebar_label: "Common Problems"
+---
+
+## Problem: Identity is unable to connect to Keycloak
+
+If you are seeing an error message such as this in your Identity service logs there is an issue with the connection
+Identity is trying to make and the Keycloak service:
+
+```
+2022-07-04 15:52:04.250 ERROR 1 --- [main] i.c.i.i.k.config.KeycloakConfiguration   : Failure #1. Unable to connect to Keycloak.
+```
+
+This can be caused by:
+
+1. The Keycloak service has not started/is not ready
+2. Identity making requests from an external IP address
+
+### Solution 1: The Keycloak service has not started/is not ready
+
+The Keycloak service can take time to start due to the supporting systems, please provide Keycloak with adequate time
```

Comment:
This is a bit of a nit pick - but do we have any idea about how much time? Or is the guidance to monitor the logs like suggested below?

Reply:
Instead of mentioning time at all, I would direct users to monitor the logs for the lines you've listed below.